symposion/proposals/models.py

```diff
@@ -109,19 +109,29 @@ class ProposalBase(models.Model):
     additional_speakers = models.ManyToManyField("speakers.Speaker", through="AdditionalSpeaker", blank=True)
     cancelled = models.BooleanField(default=False)
     tags = TaggableManager(blank=True)
+    cached_tags = models.TextField(blank=True, default='')
 
     def __unicode__(self):
         return self.title
 
+    def save(self, *args, **kwargs):
+        if self.pk:
+            # If already in database, update cached tags before saving
+            self.cached_tags = self.get_tags_display()
```

Comment:
If there's no PK, there can't be any tags pointing at this record yet, so the default value `''` for cached_tags is correct.